.eslintrc

```diff
@@ -151,7 +151,7 @@ rules:
   no-implicit-coercion: [1]
   no-implicit-globals: [0]
   no-implied-eval: [2]
-  no-inline-comments: [0]
+  no-inline-comments: [2]
```

Comment:
Ahem, now we have 50+ errors in other js files... I guess it was either a bad idea or you gave up too soon.